compliance/azure/azure_idle_compute_instances/README.md

```diff
@@ -0,0 +1,27 @@
+## Azure Idle Compute Instances
+ 
+### What it does
+This policy checks all the instances in the Azure Subscription for the average CPU usage over the the last hour.  If the usage is less than the user provided CPU percentage threshold then the virtual machines are recomended for removal, and the user is emailed.
```

Comment:
use the same terminology used in the template description and approval.  review the terms used for the actions and pick one,  the policy and readme include removal, termination and decommission.  use one for consistency and avoiding confusing.

Reply:
there is an extra THE in the first paragraph

Reply:
Is the metric really on the last hour.  That seems very aggressive.  Can we make it 30 days so it's consistent with the utilization policy?